### DISCUSSION TITLE
Wildcard URI purging?

### DISCUSSION THREAD

**Discussion by binaryfire** [spatie/laravel-responsecache#417] [Ideas]
Hi guys

I'm hoping to use this package with disk caching on a multi-tenant app. Tagging isn't possible with the disk driver but purging multiple urls using wildcards would be a great workaround. For example:

```
// Forget multiple
ResponseCache::forget('/some-uri/*');
```

Would it be easy to add this to the package? Or is there a way of doing this already? I imagine it'd be helpful for a lot of people.

Cheers

**Comment by omarend** [comment]
@binaryfire , did you find any workaround for this?

**Comment by binaryfire** [comment]
@omarend Unfortunately not. Hoping someone here can help. Or that I can convince someone to add it to the package :)
